src/activate/activate.c

```diff
@@ -356,7 +362,8 @@ static int parse_argv(int argc, char *argv[]) {
                 { "listen",      required_argument, NULL, 'l'           },
                 { "accept",      no_argument,       NULL, 'a'           },
                 { "setenv",      required_argument, NULL, 'E'           },
-                { "environment", required_argument, NULL, 'E'           }, /* legacy alias */
+                /* legacy alias */
```

Comment:
it actually belongs to the next line (the comment has been moved before the line it refers to).
By moving these comments above the line we save horizontal space, since the tables are already quite qide.

Reply:
No, please keep it at the end of previous line. As a separate line, it's much less clear what this is about.

Reply:
done here: https://github.com/systemd/systemd/pull/11690/commits/1dd1cf7907173692878a64959406ccf69e6b4704